src/main/scala/wdl4s/wdl/WdlWorkflow.scala

```diff
@@ -47,7 +49,16 @@ object WdlWorkflow {
     new WdlWorkflow(name, workflowOutputsWildcards, wdlSyntaxErrorFormatter, meta, parameterMeta, ast)
   }
 
-  def buildWomGraph(wdlWorkflow: WdlWorkflow): Graph = ???
+  def buildWomGraph(wdlWorkflow: WdlWorkflow): Graph = {
+    val graphNodes = wdlWorkflow.calls.foldLeft(Set.empty[GraphNode])({
+      case (currentNodes, call) => currentNodes ++ call.womGraphInputNodes + call.womCallNode
```

Comment:
The type system makes this annoying, the subtypes don't readily unify to the expected `GraphNode` LUB 😢